python/simulation.cpp

```diff
@@ -97,7 +97,21 @@ class simulation_shim {
     }
 
     arb::time_type run(arb::time_type tfinal, arb::time_type dt) {
-        return sim_->run(tfinal, dt);
+        arb::time_type tepoch = 0;
```

Comment:
`epoch` is an internal moniker for the timespan `t_min_delay/2`, for the sake of clarity, could you find another name?